XenModel/XenAPI-Extensions/VM.cs

```diff
@@ -729,24 +734,16 @@ public string GetVirtualisationWarningMessages()
                     }
                     else
                         return Messages.PV_DRIVERS_OUT_OF_DATE_UNKNOWN_VERSION;
-
-                default:  // VirtualisationStatus.PV_DRIVERS_NOT_INSTALLED:
-                    return Messages.PV_DRIVERS_NOT_INSTALLED;
             }
+
+            return HasNewVirtualisationStates ? Messages.VIRTUALIZATION_STATE_VM_MANAGEMENT_AGENT_NOT_INSTALLED : Messages.PV_DRIVERS_NOT_INSTALLED;

```

Comment:
I think this will give the wrong result when it's used in EvacuateHostDialog, where only I/O drivers are needed. I'm not quite sure what the right solution is: do we need to split it into more cases?